python/phy_layer.py

```diff
@@ -374,17 +391,34 @@ def initialize_usrp_sink(
         carrier_freq,
         samp_rate,
         tx_packet_length_key,
+        devices=None,
     ):
         self.logger.info("Creating USRP sink ...")
+
+        usrp_extra_stream_config = {}
+        if devices is not None:
+            # Serial USRPs might be a B210, which needs some extra config
+            # devices = tacmac.uhd_configuration.find_devices(hint=rx_device_addr)
+            self.logger.debug(f"Checking types for {tx_device_addr}")
+            for address_string in tx_device_addr.split(","):
+                for device in devices:
+                    if "serial" in device and device["serial"] == address_string.split("=")[1]:
```

Comment:
This line may benefit from a more pythonic approach.
`if device.get("serial", "") == ...`